flink-tests/src/test/java/org/apache/flink/test/checkpointing/utils/StatefulJobSavepointFrom11MigrationITCase.java

```diff
@@ -129,7 +129,7 @@ public void testCreateSavepointOnFlink11WithRocksDB() throws Exception {
 
 		executeAndSavepoint(
 				env,
-				"src/test/resources/stateful-udf-migration-itcase-flink1.1-savepoint-rocksdb",
+				"src/test/resources/stateful-udf-migration-itcase-flink1.1-rocksdbsavepoint",
```

Comment:
I this supposed to be `rocksdb-savepoint`? Otherwise shouldn't be detected by `*-savepoint`.